newIDE/app/src/UI/DragAndDrop/DragSourceAndDropTarget.js

```diff
@@ -43,6 +44,12 @@ export const makeDragSourceAndDropTarget = <DraggedItemType>(
   reactDndType: string
 ): ((Props<DraggedItemType>) => React.Node) => {
   const sourceSpec = {
+    canDrag(props: Props<DraggedItemType>, monitor: DragSourceMonitor) {
+      const item = monitor.getItem();
+      const canDrag = props.canDrag || null;
+      if (canDrag) return canDrag(item);
+      else return true;
```

Comment:
nitpicking: useless else

Reply:
Here, the function canDrag requires a boolean return type, so removing it results in flow error.

Reply:
I mean we can remove the else keyword :)